src/components/pages/WritingSub/RenderWritingSub.js

```diff
@@ -36,12 +36,15 @@ export const RenderWritingSub = props => {
         handleOk={() => {
           setModalVisible(false);
         }}
-        instructions={modalText || modalInstructions.writingSub}
+        instructions={modalText}
         style={{ fontSize: '1.5rem' }}
       />
       <div className="writing-sub-container">
-        <Row className="main-row">
+        {/* <Row className="main-row">
           <p>{props.tasks.story.writingPrompt}</p>
```

Comment:
Same as the drawingPrompt, it might be useful to leave this in, but change the seed data in the database to make more sense.

Reply:
oh! you mean to put the instructions text into the seeds so that the old code would show the relevant text? I can do that